Misc/NEWS.d/next/Documentation/2018-04-08-19-09-22.bpo-25735.idVQBD.rst

```diff
@@ -0,0 +1 @@
+Added documentations for :func:`fabs`, to take only integer input values.
```

Comment:
shouldn't this say "added documentation for func factorial to indicate that returns integer values" ?

Reply:
Noted will do

Reply:
@mcscope you can check again